perun-base/src/main/java/cz/metacentrum/perun/core/api/MemberWithSponsors.java

```diff
@@ -0,0 +1,56 @@
+package cz.metacentrum.perun.core.api;
+
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * Sponsored member with list of his sponsors (for GUI)
+ *
+ * @author Jakub Hejda <Jakub.Hejda@cesnet.cz>
+ */
+public class MemberWithSponsors {
+	private RichMember member;
+	private List<User> sponsors;
+
+	/**
+	 * Constructor
+	 */
+	public MemberWithSponsors() {
+	}
+
+	/**
+	 * Constructor
+	 */
+	public MemberWithSponsors(RichMember member) {
+		this.member = member;
+	}
+
+	public Member getMember() {
+		return member;
+	}
+
+	public void setMember(RichMember member) {
+		this.member = member;
+	}
+
+	public List<User> getListOfSponsors() {
```

Comment:
This should be just ``getSponsors()`` since it is a basic getter/setter for property ``List<User> sponsors``.

Reply:
Changed.